app/uk/gov/hmrc/gform/submission/handlebars/MdtpHttpClient.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.submission.handlebars
+
+import uk.gov.hmrc.gform.wshttp.JsonHttpClient
+
+case class MdtpHttpClient[F[_]](clients: Map[MdtpServiceName, JsonHttpClient[F]]) {
+  def select(serviceName: MdtpServiceName): JsonHttpClient[F] = clients(serviceName)
+}
+
+object MdtpHttpClient
```

Comment:
Is there any reason why this object exists?

Reply:
Initially it was wrapping an Either[HttpClient, Map[MdtpServiceName, HttpClient]], which seemed to me to need some semantics to be provided by wrapping it. However, it got simplified, so maybe it's not needed now. Shall I remove it?

Reply:
I was thinking it may exists to override default companion object for some scala magic reasons. If it is not needed than I'd say it's better to remove it.

Reply:
OK